modules/fbs/fbs.module

```diff
@@ -571,12 +571,7 @@ function fbs_form_profile2_phone_validate($element, $form_state) {
   $phone = $form_state['input']['profile_provider_fbs']['field_fbs_phone']['und'][0]['value'];
 
   if (!empty($phone)) {
-    // Preg match for danish phonenumbers
-    // Pass:
-    // (+45) 35 35 35 35 ||| +45 35 35 35 35 ||| 35 35 35 35 ||| 35353535
-    // Fail:
-    // (45)35353535 ||| 4535353535.
-    $re = '/^((\(?\+45\)?)?)(\s?\d{2}\s?\d{2}\s?\d{2}\s?\d{2})$/';
+    $re = '/^([\+][0-9]{1,3}[ \.\-]?)?([\(]{1}[0-9]{1,6}[\)])?([0-9 \.\-\/]{3,20})((x|ext|extension)[ ]?[0-9]{1,4})?$/';
```

Comment:
The update to this regular expression increases complexity and drops comment which might explain what is going on. It even 

Please add a code comment which might help future readers.
If this is a regular expression which you found elsewhere the please add a link to the source.
Also note that it fails to match one previously allowed value: `(+45) 35 35 35 35`. That may leave users in a situation where they try to change something else and this now fails.